components/fhir-common/src/main/java/org/hl7/gravity/refimpl/sdohexchange/codes/EthnicityCode.java

```diff
@@ -25,7 +25,7 @@ public static EthnicityCode fromCode(String codeString) throws FHIRException {
     return Stream.of(EthnicityCode.values())
         .filter(targetEnum -> targetEnum.code.equals(codeString))
         .findFirst()
-        .orElseThrow(() -> new FHIRException(String.format("Unsupported Ethnicity code '%s'", codeString)));
+        .orElseThrow(() -> new FHIRException(String.format("Unsupported Ethnicity value '%s'", codeString)));
```

Comment:
Your updated message now depends on the external context. You method is fromCode, and you are passing code. Why does your message contain "value"? The same is for all the other exceptions.